nbtlib/tag.py

```diff
@@ -489,9 +490,33 @@ def match(self, other):
             for item, other_item in zip(self, other)
         )
 
+    def get(self, key, default=None):
+        if isinstance(key, nbtlib.Path):

```

Comment:
I dont think the `if` here is useful.
and the  `super()` do not have `get` at all?

Reply:
Wow I completely overlooked these methods, I pasted them from the compound class but forgot to change them :upside_down_face:

Reply:
Just fixed it c491c15